picoscope/ps4000a.py

```diff
@@ -228,11 +228,11 @@ def _lowLevelSetChannel(self, chNum, enabled, coupling, VRange, VOffset,
                                        c_int16(enabled), c_enum(coupling),
                                        c_enum(VRange), c_float(VOffset))
         self.checkResult(m)
-
-        m = self.lib.ps4000aSetBandwidthFilter(c_int16(self.handle),
-                                               c_enum(chNum),
-                                               c_enum(BWLimited))
-        self.checkResult(m)
+        if self.model.startswith('4444'):  # Only for model 4444
```

Comment:
```suggestion
        # Only for model PS4444
        # See discussion: https://github.com/colinoflynn/pico-python/pull/171
        if self.model.startswith('4444'):  # Only for model 4444
```

I'm not too big of a fan of inline comments.

~~Could you please also add a release note ~~

Lol we have no changelog.

Reply:
I'm fine with either way. For some really short explanations I prefer inline comments in order to shorten the code. Thanks for your insight, it's good to hear other people's opinion.